Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build the manager binary
```

Comment:
why do we need a new Dockerfile?

Reply:
See above - part of the project generation

Reply:
Not clear then if we can just get rid of it or if we need to keep it around